src/components/Mana/Mana.tsx

```diff
@@ -3,14 +3,16 @@ import classNames from 'classnames'
 import { Network } from '@dcl/schemas/dist/dapps/network'
 import { Header, HeaderProps } from '../Header/Header'
 import './Mana.css'
+import { Popup } from '../Popup/Popup'
```

Comment:
Would you mind moving this import 1 line above? We usually sort the local imports by the following rule:
> Local imports, starting with imports referenced as modules and then sorted by distance

This would be a good sorting example:

```js
import React from 'react'
import helpers from 'helpers'
import b from 'folder/b'
import c from '../../c'
import a from '../a'
import utils from './utils'
import 'MyCss.css'
```